### DISCUSSION TITLE
disable timer in photo mode

### DISCUSSION THREAD

**Discussion by tomas123** [gopro/labs#509] [Q&A] [answered]
in photo mode (mP - mode Photo / mPN - mode Photo Night)
every start command like !S ,  !SQ , !0S etc **activate** the 3s photo timer in photo settings. 
How can I disable the timer (set Timer off)?

I know that the timer makes sense if someone triggers the QR code !S directly ;-)

GoPro Hero 11

**Comment by dnewman-gpsw** [comment, marked as answer]
While the timer briefly shows, it should be truncated to about a second.  This has been the case on Labs since it first intro on HERO8.  For some reason, long forgotten, truncating the timer was easier than removing it.
